File: sqlite3/python/delete/sqlite3_delete.py
#! /usr/bin/python3
# -*- coding: utf-8 -*-
#
#	update/sqlite_delete.py
#
#					Sep/06/2016
#
# --------------------------------------------------------
import	sys
import sqlite3
#
sys.path.append ('/var/www/data_base/common/python_common')
from sql_manipulate import sql_delete_proc
#
# --------------------------------------------------------
sys.stderr.write ("*** 開始 ***\n")
#
file_in = sys.argv[1]
key_in = sys.argv[2]
print ("%s" % key_in)
#
conn = sqlite3.connect (file_in)
conn.text_factory=str
#
cursor = conn.cursor ()
#
sql_delete_proc	(cursor,key_in)
#
conn.commit ()
#
cursor.close ()
conn.close ()
#
sys.stderr.write ("*** 終了 ***\n")
#
# --------------------------------------------------------

File: sqlite3/ruby/delete/sqlite3_delete.rb
#! /usr/bin/ruby
# -*- coding: utf-8 -*-
#
#	sqlite3_delete.rb
#
#					Feb/15/2016
#
require 'sqlite3'
#
load '/var/www/data_base/common/ruby_common/sql_manipulate.rb'
#
# ------------------------------------------------------------
puts "*** 開始 ***"
#
file_sqlite3 = ARGV[0]
#
db = SQLite3::Database.new(file_sqlite3)
#
key_in = ARGV[1]
#
puts key_in
#
sss=Sql_manipulate.new
#
sss.delete_proc(db,key_in)
#
puts "*** 終了 ***"
#
# ------------------------------------------------------------
